open_bounties/0062 Account-based toml Lookup.md

```diff
@@ -0,0 +1,44 @@
+---
+ID: "0062"
+Discussion: https://github.com/XRPLBounties/Proposals/discussions/0062
+Category: XRPL.org tools
+Status: Open
+Amount: $2,500
+---
+
+
+# Proposal Title
+
+Account-based toml Lookup Tools
+
+**Category:**
+XRPL.org tools
+
+
+**Amount:** 
+$2,500
+
+## Overview
+
+The xrp-ledger.toml checker on XRPL.org (https://xrpl.org/xrp-ledger-toml-checker.html) is a handy tool for looking up what accounts are claimed by a specific domain (including domain verification), but it could be convenient to be able to do it the other way around and check to see if an account really is claimed by the domain it has set in its Domain field.
+
+So in addition to being able to put in a domain and verify the toml from there, you could put in an account to verify that account's domain: if it has a Domain field, you look up to see if that domain serves a toml file and if that toml claims ownership of the account.
+
+
+Detailed steps requirement: 
+1. Account-based toml Lookup Script:  Write an  Account-based toml lookup tool script similar to the existing domain-based script (https://github.com/XRPLF/xrpl-dev-portal/blob/master/assets/js/xrp-ledger-toml-checker.js), and add appropriate tests.
+2. XRPL.org Web Changes : Integrate the script into https://xrpl.org/xrp-ledger-toml-checker.html with an additional input box and button for account-based lookup, make HTML/UI changes necessary to show two input boxes (one domain-based, one account-based) and connect the written script from milestone 1 to the page
```

Comment:
I'd recommend splitting up this list into nested bullets so people can "check them off" as they work on them.